cmd/orb-server/startcmd/start.go

```diff
@@ -345,6 +345,7 @@ func startOrbServices(parameters *orbParameters) error {
 		apspi.WithProofHandler(mockProofHandler(func(anchorCredID string, _, endTime time.Time, proof []byte) error {
 			return monitoringSvc.Watch(anchorCredID, endTime, proof)
 		})),
+		// apspi.WithWitnessInvitationAuth(inviteWitnessAuth),
 		apspi.WithWitness(vct.New(parameters.vctURL, vcSigner, vct.WithHTTPClient(httpClient))),
 		// apspi.WithFollowerAuth(followerAuth),
```

Comment:
Did you intend to leave these commented out lines?

Reply:
Yes, these are placeholders that should be implemented in future PRs. I just want to make sure that we don't forget about them.